io.sourceforge.pentobi.yml

```
@@ -0,0 +1,28 @@
+app-id: io.sourceforge.pentobi
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: pentobi
+rename-icon: pentobi
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+
+modules:
+  - name: pentobi
+    buildsystem: cmake
```

Cmake-ninja doesn't work?
```suggestion
    buildsystem: cmake-ninja
```

---

No it fails due to cyclic dependencies.